packages/esm-patient-chart-app/src/offline.ts

```diff
@@ -0,0 +1,95 @@
+import { NewVisitPayload, Visit } from '@openmrs/esm-api';
+import {
+  getStartedVisit,
+  getSynchronizationItems,
+  messageOmrsServiceWorker,
+  queueSynchronizationItem,
+  saveVisit,
+  setupOfflineSync,
+  VisitMode,
+  VisitStatus,
+  generateOfflineUuid,
+  subscribeConnectivity,
+} from '@openmrs/esm-framework';
+import { useEffect } from 'react';
+
+interface OfflineVisit extends NewVisitPayload {
+  uuid: string;
+}
+
+const visitSyncType = 'visit';
+
+export function setupCacheableRoutes() {
+  messageOmrsServiceWorker({
+    type: 'registerDynamicRoute',
+    pattern: '.+/openmrs/ws/fhir2/R4/Patient/.+',
```

Comment:
What I was wondering: Do we need the `/openmrs/` part in here? Wouldn't `.+/ws/fhir2/...` be sufficient? See also line 30 (consistency). And I think it may be more "reliable" (i.e., if this prefix / path changes the actual module is still named correctly).

Reply:
You are right, that was just an oversight. Thanks for mentioning it!